stable/nginx-lego/templates/default-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-default
```

Comment:
Can we change these identifiers to be default-backend instead of default. I'd like it to be more clear if possible.